hypothesis-python/src/hypothesis/_settings.py

```diff
@@ -595,66 +598,37 @@ class Statistics(IntEnum):
     always = 2
 
 
-class Verbosity(object):
+@unique
+class Verbosity(IntEnum):
+    quiet = 0
+    normal = 1
+    verbose = 2
+    debug = 3
+
+    @staticmethod
+    def _get_default():
+        var = os.getenv('HYPOTHESIS_VERBOSITY_LEVEL')
+        if var is not None:  # pragma: no cover
+            note_deprecation(
```

Comment:
Hmm. This is The Right Thing To Do, but it's one I'm going to find personally very annoying, as this is the one that I want to vary orthogonally to everything else most often.

I think this might need a new pytest flag to turn it on if we're dropping this (which obviously doesn't have to be in this pull request)

Reply:
We should also, uh, document the pytest plugin - it's currently mentioned in a few places, but nowhere do we list the options expected the behaviour.  I'll open an issue for that, and a verbosity option, after this is merged.